### DISCUSSION TITLE
How can i move windows with mouse only, without hyprbar

### DISCUSSION THREAD

**Discussion by isshin1** [hyprwm/Hyprland#8728] [Q&A]
sway allows certain windows like pip to move with mouse only , which i really appreciate. Is there a way to replicate it on hyprland

**Comment by vaxerski** [comment]
mod + click?

**Reply by isshin1** [reply]
> mod + click?

thanks for replying, but like i said, with mouse left click hold only, no keyboard involved.
now that i think about it, i have seen this behaviour on sway with pip windows only, not sure whats the window property to distinguish it.
but it would be a very nifty feature to have

**Reply by vaxerski** [reply]
likely xdg shell move and resize operations, we dont implement those

**Comment by BSFGP** [comment]
You can try assigning window movement to one of the additional mouse buttons (if there are any you don't use)

**Reply by isshin1** [reply]
yea that option is always there, but those buttons are just uncomfortable to , mostly coz of my hand shape

**Comment by vincentaxhe** [comment]
I use kmonad to bind super key to two side buttons of mouse, the buttons originly are 'pageup' and 'pagedown', when press one of them and in 1000 ms press another will triger super key.
```
(defalias
 tal ( layer-toggle mainmod )
 lm	(tap-hold-next 1000 pgup @tal :timeout-button pgdn)
 rm	(tap-hold-next 1000 pgdn @tal :timeout-button pgup)
)
  #|-------------------------sourceLayer------------------------------- |#
(defsrc
  pgup pgdn
)
  #|-------------------------base-------------------------------------- |#
(deflayer base
  @lm @rm
)
  #|-------------------------mainmod------------------------------------ |#
(deflayer mainmod
  lmet rmet
)
```
